.github/workflows/beam_PreCommit_Java_Debezium_IO_Direct.yml

```diff
@@ -85,13 +85,19 @@ jobs:
           github_job: ${{ matrix.job_name }} (${{ matrix.job_phrase }})
       - name: Setup environment
         uses: ./.github/actions/setup-environment-action
+        with:
+          java-version: |
+            17
+            8
```

Comment:
We now use Java 11 to build Beam (with Java 8 byte code compatibility). So change 8 -> 11

Reply:
Thank you! Adjusted to 11